osquery/tables/system/linux/mounts.cpp

```diff
@@ -25,10 +25,18 @@ QueryData genMounts(QueryContext& context) {
     return {};
   }
 
+  std::vector<std::string> mnt_type_blacklist = {
```

Comment:
Can we break this out of the method and make it const?

```
const std::vector<const char*> kMntTypeBlacklist = {
  "autofs",
};
```

Reply:
Yes, this makes sense. Looking at some other files it looks like the convention is to keep it in this file but move it immediately under the \#includes. I'll make that change and sync it up when I get the other change below fixed and tested.

Reply:
This is done, but instead of a vector this has been implemented as a C style array to work with the `const char*`  `ent->mnt_type` mentioned in the other conversation in this pull request.